explorer/client/src/utils/static/mock_data.json

```diff
@@ -644,7 +644,76 @@
                 }
             }
         },
-
+        {
+            "id": "standaloneObjectABC",
+            "category": "object",
+            "owner": { "SingleOwner": "player2" },
+            "version": "87",
+            "readonly": "true",
+            "objType": "0x2::coin::Coin<0x2::abc::ABC>",
+            "data": {
+                "contents": {
+                    "id": {},
+                    "balance": "1"
+                }
+            }
+        },
+        {
+            "id": "standaloneObjectDEF",
+            "category": "object",
+            "owner": { "SingleOwner": "player2" },
+            "version": "87",
+            "readonly": "true",
+            "objType": "0x2::coin::Coin<0x2::def::DEF>",
+            "data": {
+                "contents": {
+                    "id": {},
```

Comment:
If we assign faked object IDs here, these values are not used because `checkIsPropertyType` restricts properties to be either strings or numbers while IDs are entered as objects. This is why a value is not used here. We are in the process of updating the testing approach to replace the mock data JSON files with @Jordan-Mysten currently putting together a strategy.